src/bundle/Resources/config/adminui.yml

```diff
@@ -0,0 +1,9 @@
+services:
+
+    ibexa.automated_translate.adminui.component.javascripts:
```

Comment:
Hmm, naming - `automated_translate` has some grammar issue, but more importantly, where's the class for that service? I see only parent definition.

Reply:
similar as https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/config/services/dashboard.yaml#L47